### PR TITLE
Fix SQL migration conflict and coverage gaps

### PR DESCRIPTION
**Pull request by nguyer**
We had two PRs get merged separately that used the same migration number. This PR also fixes two small gaps in unit test coverage.

### CODE REVIEW

File: test/e2e/e2e.log
Comment (awrichar) [inline test/e2e/e2e.log:1]: Don't think this was supposed to be checked in?

File: test/e2e/multiparty/fabric_contracts.go [hunk 1]
```diff
@@ -163,8 +163,8 @@ func (suite *FabricContractTestSuite) TestE2EContractEvents() {
 	assert.NoError(suite.T(), err)
 	assert.Equal(suite.T(), assetName, res.(map[string]interface{})["name"])
 
-	suite.testState.client1.DeleteContractListener(suite.T(), subs[0].ID)
-	subs = suite.testState.client1.GetContractListeners(suite.T(), suite.testState.startTime)
-	assert.Equal(suite.T(), 0, len(subs))
```
Comment (awrichar) [inline test/e2e/multiparty/fabric_contracts.go:168]: Did you mean to leave this commented out?

Comment (nguyer) [reply test/e2e/multiparty/fabric_contracts.go:168]: No 🤦

File: test/e2e/e2e.log
Comment (nguyer) [reply test/e2e/e2e.log:1]: 🧹

**Review by shorsher — APPROVED** [review]
🚀

**Pull request merged**